package/MDAnalysis/core/groups.py

```diff
@@ -119,6 +119,10 @@ def _unpickle(u, ix):
     return u.atoms[ix]
 
 
+def _unpickle2(u, ix, cls):
```

Comment:
Just one little nitpick, would you be able to document in a quick comment why the two are necessary for someone following along later?